genai_cookbook/nbs/2-fundamentals-unstructured-chain.md

```diff
@@ -1,6 +1,6 @@
-## Retrieval, augmentation, and generation (aka RAG Chain)
+## Retrieval, augmentation, and generation (aka RAG Agent)
 
-Once the data has been processed by the data pipeline, it is suitable for use in the RAG application. This section describes the process that occurs once the user submits a request to the RAG application in an online setting. The series, or *chain* of steps that are invoked at inference time is commonly referred to as the RAG chain.
+Once the data has been processed by the data pipeline, it is suitable for use in the retreival tool. This section describes the process that occurs once the user submits a request to the Agent application in an online setting. The series, or *chain* of steps that are invoked at inference time is commonly referred to as the Agent loop.
```

Comment:
```suggestion
Once the data has been processed by the data pipeline, it is suitable for use in a retriever tool. This section describes the process that occurs once the user submits a request to the agent application in an online setting.
```

Not sure if "Agent loop" is actually a standard term, does it appear in LC/LG docs etc?